client/src/Components/Layouts/LangingPge/LangingPge.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { StaticRouter } from "react-router-dom";
+
+import LandingPage from "./index";
+
+test("LandingPage matches snapshot", () => {
+  const context = {};
+
+  const component = renderer.create(
+    <StaticRouter context={context}>
+      <LandingPage />
+    </StaticRouter>,
+  );
+  expect(component).toMatchSnapshot();
```

Comment:
This fails when I run locally

![screen shot 2018-11-26 at 12 40 06](https://user-images.githubusercontent.com/28838993/49014580-86847f80-f178-11e8-98ea-5c86df8429a6.png)

Reply:
@arrested-developer 
I ran that locally and the test passed, could you check that again, please